sources/packages/web/src/assets/css/base.scss

```diff
@@ -59,38 +59,31 @@ html * {
   padding: 0.5rem !important;
 }
 
-/** Moving the dimension and position properties of status badge to common class **/
-.status_badge {
-  border-width: 1px;
-  padding: 15px;
-  border-style: solid;
-  border-radius: 30px;
-  position: static;
-  font-weight: 700;
+// Specific styles for the outlined v-chip.
+.v-chip--variant-outlined {
+  border: 2px solid !important;
+  font-weight: 700 !important;
+  background-color: white !important;
```

Comment:
maybe we can add the color `white` or the color code to scsss variable

Reply:
I can create something as `$badge-`backgroung` since all the other ones calling it `$badge-something`.

Reply:
Changed, please take a look.